### ISSUE TITLE
Show confirmation modal when user initiates destructive operations (resource/tag removal)

### ISSUE THREAD

**Issue by melvin4u445** [ARK-Builders/ARK-Navigator#98]
The delete button deletes files without any confirmation. It's not that difficult to accidentally press and hold the delete button while the user isn't looking at the phone. 
Should we show a modal instead and ask the user to confirm?

**Comment by melvin4u445** [comment]
Agreed with @kirillt that it would make sense to show an "Are you sure you want to delete this" confirmation modal along with a "Don't show again" checkbox for those who prefer the current way.

**Comment by kirillt** [comment]
Also long-press on tag chip in Gallery mode (removing tag without opening the modal with "pen" button).
